scala/scala-impl/src/org/jetbrains/plugins/scala/actions/NewPredefinedSbtFileAction.scala

```diff
@@ -0,0 +1,31 @@
+package org.jetbrains.plugins.scala.actions
+
+import com.intellij.ide.IdeView
+import com.intellij.ide.fileTemplates.actions.CreateFromTemplateActionBase
+import com.intellij.ide.fileTemplates.{FileTemplate, FileTemplateManager}
+import com.intellij.openapi.actionSystem.{CommonDataKeys, DataContext}
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.Messages
+import com.intellij.openapi.util.NlsActions
+import com.intellij.psi.PsiDirectory
+import org.jetbrains.plugins.scala.ScalaBundle
+import org.jetbrains.plugins.scala.extensions.PsiNamedElementExt
+
+import javax.swing.Icon
+
+abstract class NewPredefinedSbtFileAction(@NlsActions.ActionText title: String, @NlsActions.ActionDescription description: String, icon: Icon, fileName: String) extends CreateFromTemplateActionBase(title, description, icon) {
```

Comment:
this line with class parameters is too long. Each parameter should be in a separate line.  The same like in methods.
![Screenshot 2024-05-15 at 17 55 00](https://github.com/JetBrains/intellij-scala/assets/33667003/857d56c0-3190-4ba2-870d-036315bb956d)